src/qt/optionsmodel.cpp

```diff
@@ -252,14 +252,14 @@ void OptionsModel::SetPruneEnabled(bool prune, bool force)
     }
 }
 
-void OptionsModel::SetPruneTargetGB(int prune_target_gb, bool force)
+void OptionsModel::SetPruneTargetGB(int prune_target_gb)
```

Comment:
In commit "refactor: Drop `bool force' parameter" (0849b512916be3411db9d1efa447f2c12552ee0d)

Not a big deal, but I don't think the force argument should be dropped from this function. The call site in InitializePruneSetting is clearer if it has to explicitly set `force` to true, and it seems inconsistent and unsafe for one optionsmodel `SetPrune` method to soft-set by default while the other one force sets. The need to soft or force set depends on the context of the SetPrune call, and isn't something that seems right to hardcode here.

Reply:
Done.